docs/content/guides/observability/ui/setup.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: Set up the Gloo UI
-weight: 
-description: Install the Gloo UI to get an at-a-glance view of the configuration, health, and compliance status of your Gloo Gateway setup and the workloads in your cluster. 
+weight: 10
+description: Install the Gloo UI and start gaining insights into the configuration and health of your Gloo Gateway setup and the workloads in your cluster. 
 ---
 Install the Gloo UI to get an at-a-glance view of the configuration, health, and compliance status of your Gloo Gateway setup and the workloads in your cluster. 
 
 To learn more about the features of the Gloo UI, see [About the Gloo UI]({{< versioned_link_path fromRoot="/guides/observability/ui/explore#about-the-gloo-ui" >}}).
 
 ## Before you begin
 
-Install GG
+Follow the steps to [install Gloo Gateway with the Enterprise Edition]({{< versioned_link_path fromRoot="/installation/enterprise/" >}}). 
```

Comment:
I can't comment on the section because it wasn't updated, but for the paragraph right after "Set up the Gloo UI" it says:

> Use these instructions to install the Gloo UI in the same cluster as Gloo Gateway.

Does this mean the same cluster as the control plane? although I guess the control plane would be in each cluster, even in a multicluster env, so maybe that distinction doesn't matter. 

I guess I was wondering if the UI does multicluster and if so, which cluster you install it in.

Reply:
I changed the sentence. I don't want to highlight multicluster capabilities here as Gloo Edge does not really support multicluster.